cmd/kubeadm/app/cmd/init.go

```diff
@@ -281,7 +281,7 @@ func (i *Init) Run(out io.Writer) error {
 
 	// Create the default node bootstrap token
 	tokenDescription := "The default bootstrap token generated by 'kubeadm init'."
-	if err := nodebootstraptokenphase.UpdateOrCreateToken(client, i.cfg.Token, false, i.cfg.TokenTTL, kubeadmconstants.DefaultTokenUsages, tokenDescription); err != nil {
+	if err := nodebootstraptokenphase.UpdateOrCreateToken(client, i.cfg.Token, false, i.cfg.TokenTTL, kubeadmconstants.DefaultTokenUsages, []string{}, tokenDescription); err != nil {
```

Comment:
The default kubeadm token should have a prefix of `system:bootstrappers:kubeadm:default-node` or something.
Please add that to constants.go
We should also migrate the RBAC-creation code to create a ClusterRoleBinding depending on version